Systems/c172p-damage.xml

```diff
@@ -243,38 +243,39 @@
             <test logic="AND" value="1.0">
                 /velocities/airspeed-kt GT damage/limits-vne-high
             </test>
+            
             <test logic="AND" value="1.0">
                 /velocities/airspeed-kt GT damage/limits-vne-medium
                 damage/roll-moment GT 4000.0
-                wing-damage/right-wing GT 0.0
+                wing-damage/right-wing LT 1.0
```

Comment:
> We can break the left one even if the right one is OK.

But now you have changed it to`wing-damage/right-wing LT 1.0`. So that means that when the right wing is broken, you cannot break the left wing anymore unless you go faster than `limits-vne-high`. Is that on purpose?